core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -233,25 +233,25 @@ public void applyTo(TableMetadata.Builder metadataBuilder) {
   }
 
   class SetSnapshotRef implements MetadataUpdate {
-    private final String name;
+    private final String refName;
     private final Long snapshotId;
     private final SnapshotRefType type;
     private Integer minSnapshotsToKeep;
     private Long maxSnapshotAgeMs;
     private Long maxRefAgeMs;
 
-    public SetSnapshotRef(String name, Long snapshotId, SnapshotRefType type, Integer minSnapshotsToKeep,
+    public SetSnapshotRef(String refName, Long snapshotId, SnapshotRefType type, Integer minSnapshotsToKeep,
                           Long maxSnapshotAgeMs, Long maxRefAgeMs) {
-      this.name = name;
+      this.refName = refName;
       this.snapshotId = snapshotId;
       this.type = type;
       this.minSnapshotsToKeep = minSnapshotsToKeep;
       this.maxSnapshotAgeMs = maxSnapshotAgeMs;
       this.maxRefAgeMs = maxRefAgeMs;
     }
 
-    public String name() {
-      return name;
+    public String refName() {
```

Comment:
Probably better not to rename this, since it requires touching two extra files.

Reply:
Reverted.